app/controllers/movies_controller.rb

```diff
@@ -1,7 +1,8 @@
 class MoviesController < ApplicationController
+  PER_PAGE = 10
   def index
-    # 取得対象ジャンル
     genre_list = %W(Basic Git HTML&CSS Ruby Ruby on Rails)
     @movies = Movie.where(genre: genre_list)
+    @movies = Movie.page(params[:page]).per(PER_PAGE)
```

Comment:
以下のようにつなげないと，絞り込みができなくなってしまいます。
例えば今の書き方ですと，PHP教材も入ってしまいます。`texts`の場合も同様です！

```suggestion
    @movies = Movie.where(genre: genre_list).page(params[:page]).per(PER_PAGE)
```

Reply:
ご指摘ありがとうございます！
本当ですね！このqiita記事に載ってました！直しました！https://qiita.com/nysalor/items/77b9d6bc5baa41ea01f3